game/static/game/css/board.css

```
@@ -3345,3 +3345,101 @@ body.blindfold-mode .capture-ring {
+}
+
+.resume-game-floating-btn:active {
+    transform: translate(-50%, 0);
+}
+
+.resume-game-floating-btn.hidden {
+    display: none !important;
+    pointer-events: none;
+}
+
+@keyframes pulseResume {
```

Stylelint (17.14.0)

[error] Expected keyframe name "pulseResume" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)